interpreter/computer_use/loop.py

```diff
@@ -487,6 +487,8 @@ async def tool_output_callback(result: ToolResult, tool_id: str):
 
             print_markdown("\nWe'll email you shortly. ✓\n---\n")
             continue
+        elif user_input=="":
```

Comment:
Thanks @ehlowe 

Can you please include a check for whitespace. A space character leads to the same error

Reply:
Yes, just fixed it